enhancements/hide-container-mountpoints.md

```diff
@@ -0,0 +1,339 @@
+---
+title: hide-container-mountpoints
+authors:
+  - "@lack"
+reviewers:
+  - "@haircommander"
+  - "@mrunalp"
+  - "@umohnani8"
+approvers:
+  - "@haircommander"
+  - "@mrunalp"
+  - "@umohnani8"
+api-approvers:
+  - None
+creation-date: 2021-01-18
+last-updated: 2021-05-19
+tracking-link:
+  - https://issues.redhat.com/browse/CNF-5326
+---
+
+# Hide Container Mountpoints
+
+## Summary
+
+The current implementation of Kubelet and CRI-O both use the top-level
+namespace for all container and Kubelet mountpoints. However, moving these
+container-specific mountpoints into a private namespace reduced systemd
+overhead with no difference in functionality.
+
+## Motivation
+
+Systemd scans and re-scans mountpoints many times, adding a lot to the CPU
```

Comment:
nit: Systemd -> systemd.

Reply:
Even at the beginning of a sentence?

Reply:
Yeah, since it is a name :) See https://www.freedesktop.org/wiki/Software/systemd/ #Spelling section.